network/client.go

```diff
@@ -0,0 +1,110 @@
+package network
+
+import (
+	"errors"
+	"fmt"
+	"strconv"
+	"sync"
+	"time"
+
+	"github.com/dedis/crypto/config"
+)
+
+// Client is an interface which is a simpler version of a Router. The main use
+// for a Client is to directly Send something and get a result back. It is used
+// intensively by Services to have a easy external API.
+// Two implementations are done: TcpClient to use for applications and
+// deployement,and localClient to use for local testing alongside with
+// LocalRouter.
+// NOTE: This interface is likely to be removed to be replaced by a full
+// pledged REST HTTP Api directly connected to the sda/services.
+type Client struct {
+	connector func(own, remote *ServerIdentity) (Conn, error)
+}
+
+func newClient(c func(own, remote *ServerIdentity) (Conn, error)) *Client {
+	return &Client{c}
+}
+
+var baseID uint64
+var baseIDLock sync.Mutex
+
+var timeoutResponse = 10 * time.Second
```

Comment:
Ah, I see you chnage it in the tests (which makes sense)... Sorry for the noise.